### DISCUSSION TITLE
adaption to file:// protocol

### DISCUSSION THREAD

**Discussion by volker-leins** [Pagefind/pagefind#81] [Ideas]
Great Work! It runs completely local with dev server but not file:// protocol due to import statements and CORS. An adaption to use only <script> for local pure browser based apps would be a great enhancement.

**Comment by bglw** [comment]
Unfortunately with the mix of JS files and WebAssembly that Pagefind requires in the browser I don't see an easy way to enable this sorry — especially on the WebAssembly side that will always be plagued by CORS errors outside of a server setting.
